File: Mobile/Albi/app/src/main/java/albi/bme/hu/albi/model/Flat.kt
package albi.bme.hu.albi.model

import java.io.Serializable

class Flat(var _id: String = "",
           var userID: String = "",
           var price: String = "" ,
           var numberOfRooms: String = "",
           var description: String = "",
           var email: String = "",
           var phone_number: String = "",
           var zipCode: String = "",
           var city: String = "",
           var address: String = "",
           var forSale: Boolean = false) : Serializable{

    var imageNames: ArrayList<String>? = ArrayList()

}

File: Frontend/Albi/src/app/flat.ts
import { Image } from './image';

export class Flat {
  _id: String;
  userID: String;
  price: String;
  numberOfRooms: String;
  description: String;
  email: String;
  phone_number: String;
  zipCode: String;
  city: String;
  address: String;
  forSale: Boolean;
  images: Image[];
  firstImage: Image;
  noImageFound = true;


  constructor() {
    this.images = [];
    this.firstImage = new Image();
  }
}
